lib/vanagon/engine/base.rb

```diff
@@ -57,7 +57,7 @@ def ship_workdir(workdir)
 
       def retrieve_built_artifact
         FileUtils.mkdir_p("output")
-        Vanagon::Utilities.rsync_from("#{@remote_workdir}/output/*", "#{@target_user}@#{@target}", "output", @platform.ssh_port)
+        Vanagon::Utilities.rsync_from("#{@remote_workdir}/output/*", "#{@target_user}@#{@target}", "output/", @platform.ssh_port)
```

Comment:
Trailing slash doesn't/shouldn't matter on the rsync receiver side, only on the sender side.